### DISCUSSION TITLE
How to send only the selected text/line in sender?

### DISCUSSION THREAD

**Discussion by gerroon** [kingToolbox/WindTerm#528] [Q&A] [answered]
Hi

Is it possible to send only the selected text or the line  in the sender tab?

**Comment by kingToolbox** [comment, marked as answer]
Unfortunately, this is not currently supported.

But you can do this:
- Copy the selected text or the line.
- Click the `+` button to create a new `Sender`.
- Paste the text and start sending.

It's a bit complicated, but it works.

**Reply by wesleyel** [reply]
Thanks for your reply!

This feature is really useful, please consider to add it. Set the shortcut to `Ctrl + Enter` will be great!
